src/app.ts

```diff
@@ -1,6 +1,71 @@
+type ResultData = {
+  id: string;
+  gender: string;
+  age: number;
+  row: number;
+};
+
+type PagingData = {
+  next?: string;
+  previous?: string;
+};
+
+let previousBtn: HTMLElement | any = document.querySelector("button[data-prevbtn]");
+let nextBtn: HTMLElement | any = document.querySelector("button[data-nextbtn]");
+let pageView: HTMLElement | any = document.querySelector("label[data-pageview]");
+let tableBody: HTMLElement | any = document.querySelector("tbody[data-sink]");
+let currentPage: number = 1;
+
+const paginate = () => {
+  previousBtn.disabled = true;
+  previousBtn.style.color = 'black';
+  previousBtn.style.backgroundColor = 'gray';
+  pageView.innerHTML = `Page: ${currentPage}`;
+  previousBtn.addEventListener('click', async () => {
+    if (currentPage > 1) {
+      currentPage--;
+    }
+    pageView.innerHTML = `Page: ${currentPage}`;
+    tableBody.innerHTML = "";
+    await loadData(currentPage);
+  });
+  nextBtn.addEventListener('click', async () => {
+    currentPage++;
+    pageView.innerHTML = `Page: ${currentPage}`;
+    tableBody.innerHTML = '';
+    await loadData(currentPage);
+  });
+};
+
+const loadData = async (page: number = 1) => {
+  const response = await fetch(
+    `https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84&page=${page}`,
+  );
+
+  if (response.status === 200) {
+    const resData = await response.json();
+    const paging: PagingData = resData?.results[0].paging;
+    const data: ResultData[] = resData?.results[0][`${currentPage}`];
+    if (paging.previous) {
+      previousBtn.disabled = false;
+      previousBtn.style.color = '#04AA6D';
+    }
+
+    for (let i = 0; i < data?.length; i++) {
+      const row = data[i];
+      let tr = document.createElement('tr');
```

Comment:
Codacy has a fix for the issue: [Replace `············let·tr·=·document.createElement('tr'` with `······let·tr·=·document.createElement("tr"`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
      let tr = document.createElement("tr");
```